src/main/java/com/blamejared/crafttweaker/impl_native/entity/ExpandPlayerEntity.java

```diff
@@ -20,82 +26,198 @@
+    
         return internal.getPortalCooldown();
     }
     
     @ZenCodeType.Method
     public static boolean drop(PlayerEntity internal, boolean p_225609_1_) {
+    
         return internal.drop(p_225609_1_);
     }
     
     @ZenCodeType.Method
     public static boolean isUser(PlayerEntity internal) {
+    
         return internal.isUser();
     }
     
     @ZenCodeType.Method
     public static void wakeUp(PlayerEntity internal) {
+    
         internal.wakeUp();
     }
     
     @ZenCodeType.Method
     public static int getSleepTimer(PlayerEntity internal) {
+    
         return internal.getSleepTimer();
     }
     
     @ZenCodeType.Method
     public static void unlockRecipes(PlayerEntity internal, ResourceLocation[] p_193102_1_) {
+    
         internal.unlockRecipes(p_193102_1_);
     }
     
     @ZenCodeType.Method
     public static boolean isSpectator(PlayerEntity internal) {
+    
         return internal.isSpectator();
     }
     
     @ZenCodeType.Method
     public static boolean isSwimming(PlayerEntity internal) {
+    
         return internal.isSwimming();
     }
     
     @ZenCodeType.Method
     public static boolean isCreative(PlayerEntity internal) {
+    
         return internal.isCreative();
     }
     
     @ZenCodeType.Method
     public static boolean isPushedByWater(PlayerEntity internal) {
+    
         return internal.isPushedByWater();
     }
     
     @ZenCodeType.Method
     public static float getLuck(PlayerEntity internal) {
+    
         return internal.getLuck();
     }
     
     @ZenCodeType.Method
     public static boolean canUseCommandBlock(PlayerEntity internal) {
+    
         return internal.canUseCommandBlock();
     }
     
     @ZenCodeType.Method
     public static void sendMessage(PlayerEntity internal, MCTextComponent text) {
+        
         internal.sendMessage(text.getInternal(), CraftTweaker.CRAFTTWEAKER_UUID);
     }
     
+    @ZenCodeType.Method
+    public static void sendStatusMessage(PlayerEntity internal, MCTextComponent text, boolean actionBar) {
+        
+        internal.sendStatusMessage(text.getInternal(), actionBar);
+    }
+    
+    /**
+     * @param mutable if true, it will return a {@link MCItemStackMutable}, instead of a copy one. Use it with caution.
+     */
+    @ZenCodeType.Method
+    public static IItemStack getCurrentItem(PlayerEntity internal, @ZenCodeType.Optional boolean mutable) {
+        
+        if(mutable) {
+            return new MCItemStackMutable(internal.inventory.getCurrentItem());
+        } else {
+            return new MCItemStack(internal.inventory.getCurrentItem());
+        }
+    }
+    
+    /**
+     * @param mutable if true, it will return a {@link MCItemStackMutable}, instead of a copy one. Use it with caution.
+     */
+    @ZenCodeType.Method
+    public static IItemStack getInventoryItemStack(PlayerEntity internal, int slotIndex, @ZenCodeType.Optional boolean mutable) {
```

Comment:
Why the additional mutable parameter?
People could just do `getInventoryItemStack(5).mutable()`

Additionally, Optional booleans don't work properly atm, have you tested this method?